server/src/models/application-files.es6

```diff
@@ -4,7 +4,8 @@ const Sequelize = require('sequelize');
 const url = require('url');
 
 const sequelizeOptions = {
-  dialect: url.parse(process.env.DATABASE_URL, true).protocol.split(':')[0]
+  dialect: url.parse(process.env.DATABASE_URL, true).protocol.split(':')[0],
+  logging: false
```

Comment:
Why is logging false here? Also aren't dialects set globally, not locally in the `.sequelize` config?